testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveSparkCompatibility.java

```diff
@@ -97,4 +96,36 @@ public void testTrinoReadingTableCreatedByNativeSpark()
 
         onSpark().executeQuery("DROP TABLE " + sparkTableName);
     }
+
+    @Test(groups = {HIVE_SPARK_BUCKETING, PROFILE_SPECIFIC_TESTS})
+    public void testSparkReadingTableCreatedByNativeTrino()
+    {
+        String sparkTableName = "test_spark_reading_trino_native_buckets_" + randomTableSuffix();
+        String trinoTableName = format("%s.default.%s", TRINO_CATALOG, sparkTableName);
+
+        onTrino().executeQuery("SET SESSION hive.experimental_parquet_optimized_writer_enabled = true");
+
+        onTrino().executeQuery(format(
+                "CREATE TABLE %s (\n" +
+                        "   a_string varchar,\n" +
+                        "   a_bigint bigint,\n" +
+                        "   an_integer integer,\n" +
+                        "   a_real real,\n" +
+                        "   a_double double,\n" +
+                        "   a_boolean boolean\n" +
+                        ")\n" +
+                        "WITH (\n" +
+                        "   format = 'Parquet'\n" +
```

Comment:
- uppercase "PARQUET"
- nit: replace  `\n` in the SQL with single space ` ` -- will make query more readable in IDE